hangman.rb

```diff
@@ -0,0 +1,125 @@
+
+require 'colorize'
+
+class Hangman
+
+ 
+
+  def initialize
+    @user_guesses = []
+    @wrong_count = 0
+    @word = pick_word 
+    puts "Answer: #{@word.join}"
+    @underlines = create_underlines
+    @ascii =  ["|     _________", 
+               "|     |/      |", 
+               "|     |",
+               "|     |",
+               "|     |",
+               "|     |",
+               "|     |",
+              ]
+
+    @man_array =  [
+                  "|     |      / \\".colorize(:red),
+                  "|     |       |".colorize(:cyan),
+                  "|     |      \\|/".colorize(:light_yellow),
+                  "|     |      (_)".colorize(:magenta)
+                  ]
+
+    play
+  end
+
+#Program picks a word for the player.
+  def pick_word
+    word_options = %w(scared selection shake shaking shallow shout silly simplest slight
+                    slip slope soap solar species spin stiff swung tales thumb zoo
+                    tobacco toy trap treated tune university vapor vessels wealth wolf)
+
+    word = word_options.sample.split(//)
+    return word
+  end
+#joins the ascii art and the underlines for the letters in the word.
+  def ascii_art
+    @ascii[7] = @underlines.join
```

Comment:
Good suggestion! I definitely need to work on my commenting skills.

The first time ascii_art is called, there's nothing there, but after that, we need to replace [7] with the new underlines (e.g. _ _ _ --> _ A _ --> C A _ --> C A T)